src/components/item/edit/EditModal.tsx

```diff
@@ -131,6 +123,48 @@ const EditModal = ({ item, onClose, open }: Props): JSX.Element => {
     onClose();
   };
 
+  // temporary solution for displaying separate dialog content
+  const renderContent = () => {
+    if (item.type === ItemType.LOCAL_FILE || item.type === ItemType.S3_FILE) {
+      return <FileForm onClose={onClose} item={item} />;
+    }
+
+    return (
+      <>
+        <DialogContent
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+          }}
+        >
+          {renderDialogContent()}
+        </DialogContent>
+        <DialogActions>
+          <CancelButton
+            id={EDIT_ITEM_MODAL_CANCEL_BUTTON_ID}
+            onClick={onClose}
+          />
+          <Button
+            // should not allow users to save if the item is not valid
+            disabled={
+              // // maybe we do not need the state variable and can just check the item
```

Comment:
```suggestion
              // maybe we do not need the state variable and can just check the item
```

Reply:
Is this comment still valid?

Reply:
probably not, but this code will disappear anyway, I'll let it be for now.